sphinx/developing/auto-cli-api.rst

```diff
@@ -0,0 +1,167 @@
+Auto CLI API
+============
+
+
+The rose command line API is auto-documented by the extension ``auto-cli-api``.
```

Comment:
is rose capitalised? (not sure what the convention is...)

Reply:
So rose is supposedly a contrived acronym but we now treat it as a name, which we don't capitalise? @matthewrmshin ...